### PR TITLE
Added sbom index and what is sbom article

### PR DESCRIPTION
**Pull request by sheesh**

### CODE REVIEW

**Comment by sheesh** [comment]
The "What is an SBOM" article is ready for review. I already had Dan Luhring and Adolfo look at it for technical guidance. 

For some reason, my directory description is not showing up the SBOM landing page. Tips appreciated. Thanks!

File: content/open-source/sbom/what-is-an-sbom.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+* **Identify software suppliers**: SBOMs can be used to identify the _suppliers_ of software components, such as a software author or the individual or entity repackaging the software for redistribution. Supplier identity is often investigated for legal or procurement reasons and there is an evolving debate about which types of distributors should be formally considered as "suppliers".    
+* **Licensing**: Software projects often pull from tens or hundreds of dependencies whose different licenses may have different restrictions on the use of the code. Being able to track the licenses of your dependencies allows you to stay informed about legal restrictions and licensing compatibility issues, and help you decide on which dependencies work best for your project needs.
```
Comment (ltagliaferri) [inline content/open-source/sbom/what-is-an-sbom.md:27]:
```suggestion
* **Licensing**: Software projects often pull from tens or hundreds of dependencies whose different licenses may have different restrictions on the use of the code. Being able to track the licenses of your dependencies allows you to stay informed about legal restrictions and licensing compatibility issues, and help you decide which dependencies work best for your project needs. 
```

File: content/open-source/sbom/what-is-an-sbom.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+## Learn more
+In this guide, you have learned about the purpose of SBOMs and why proponents see them as a critical building block for software supply chain security. You have also learned about key tools and formats used in SBOM production and consumption, and how to measure the quality of the SBOMs you generate or consume. 
+
+SBOM practices and tooling are actively evolving. To learn more about SBOMs, check out related research by Chainguard Labs, such as [What Makes a Good SBOM?](https://edu.chainguard.dev/open-source/sbom/what-makes-a-good-sbom/), [Are SBOMs Any Good? Preliminary Measurement of the Quality of Open Source Project SBOMs](https://www.chainguard.dev/unchained/are-sboms-any-good-preliminary-measurement-of-the-quality-of-open-source-project-sboms), and [Are SBOMs Good Enough for Government Work?](https://www.chainguard.dev/unchained/are-sboms-good-enough-for-government-work).
```
Comment (ltagliaferri) [inline content/open-source/sbom/what-is-an-sbom.md:62]:
```suggestion
SBOM practices and tooling are actively evolving. To learn more about SBOMs, check out related research by Chainguard Labs, such as: 
* [What Makes a Good SBOM?](https://edu.chainguard.dev/open-source/sbom/what-makes-a-good-sbom/)
* [Are SBOMs Any Good? Preliminary Measurement of the Quality of Open Source Project SBOMs](https://www.chainguard.dev/unchained/are-sboms-any-good-preliminary-measurement-of-the-quality-of-open-source-project-sboms)
* [Are SBOMs Good Enough for Government Work?](https://www.chainguard.dev/unchained/are-sboms-good-enough-for-government-work)
```

**Review by ltagliaferri — APPROVED** [review]

File: content/open-source/sbom/_index.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+---
+title: "SBOMs"
+lead: "Software Bill of Materials"
+description: "Software Bill of Materials"
+date: 2023-01-26T08:49:15+00:00
```
Comment (ltagliaferri) [inline content/open-source/sbom/_index.md:5]: To change the way the page looks

```suggestion
type: "article"
date: 2023-01-26T08:49:15+00:00
```

**Pull request merged**